glue-codes/fast-farm/src/FASTWrapper.f90

```diff
@@ -184,7 +184,7 @@ SUBROUTINE FWrap_Init( InitInp, u, p, x, xd, z, OtherState, y, m, Interval, Init
          return
       end if
       
-      call move_alloc(m%Turbine%IfW%m%FDext%V, u%Vdist_High)
+      call move_alloc(m%Turbine%IfW%m%FlowField%Grid4D%Vel, u%Vdist_High)
```

Comment:
This will work for now.  We'll storage from `m%` to parameters when we switch to pointers.

Reply:
I think we may have to leave `FlowField` in `m` because it can be modified when calling `FlowField_GetVelAcc`. When I added the BoxExceed functionality it needed a flag to indicate if a warning had already been displayed about being outside of the box. If `FlowField` is in `p` then it will have `intent(in)` in most function calls so it can't be modified. I'm open to ideas about how to deal with this since I don't think we should store state in `FlowField`.

Reply:
I agree we shouldn't store state info in `FlowField`.  Perhaps instead of issuing a warning during the simulation, we simply alert the user during `Init` that some points outside the wind box will be allowed.